src/auro-combobox.js

```diff
@@ -45,6 +45,7 @@ class AuroCombobox extends LitElement {
     this.noFilter = false;
     this.value = null;
     this.optionSelected = null;
+    this.error = false;
```

Comment:
Nit: Why was the default set to `false`? The only reference I see here is to set `this.error='true'`

I am curious as to what value pre-setting this to `false` in the constructor brings versus leaving it as `undefined`? Is this a Svelte issue?

Reply:
Wasn't actually necessary. I removed it.